src/webrtc/bandwidth.ts

```diff
@@ -0,0 +1,64 @@
+import {
+  SimulcastBandwidthLimit,
+  TrackBandwidthLimit,
+  TrackEncoding,
+} from './types';
+
+export const applyBandwidthLimitation = (
+  encodings: RTCRtpEncodingParameters[],
+  max_bandwidth: TrackBandwidthLimit,
```

Comment:
It would be nice to follow single rule for variables naming (`snake_case` vs `camelCase`). In most cases camel case seems to be used, to let's follow that rule